scripts/lua/management/lib/swagger.lua

```diff
@@ -98,23 +98,24 @@ end
 function parseRateLimit(swagger, policies)
   local rlObj = swagger["x-gateway-rate-limit"]
   rlObj = (rlObj == nil) and swagger["x-ibm-rate-limit"] or rlObj
+  local unit
```

Comment:
What does making this a local variable buy us? How is it better than it was before?